crypto/keyring/keyring.go

```diff
@@ -876,31 +877,36 @@ func (ks keystore) MigrateAll() (bool, error) {
 }
 
 // migrate converts keyring.Item from amino to proto serialization format.
-func (ks keystore) migrate(key string) (keyring.Item, bool, error) {
+func (ks keystore) migrate(key string) (*Record, bool, error) {
```

Comment:
From my perspective, it is more efficient to return `Record,bool,error`. In this case we avoid duplicated invocation of `ks.protoUnmarshalRecord`.

Reply:
OK